src/compiler/utilities.ts

```diff
@@ -1553,6 +1553,10 @@ namespace ts {
 
     /** Does the opposite of `getJSDocParameterTags`: given a JSDoc parameter, finds the parameter corresponding to it. */
     export function getParameterFromJSDoc(node: JSDocParameterTag): ParameterDeclaration | undefined {

```

Comment:
See the use of this function -- at `x` in `@param obj.x` we should return the symbol for `x`. Add a find-all-references test.
```ts
/**
 * @param obj.[|x|]
 */
function f(obj) {
    obj.[|x|];
}
```